packages/calypso-build/babel/default.js

```diff
@@ -16,7 +16,7 @@ module.exports = ( api, opts ) => ( {
 		[
 			require.resolve( '@babel/preset-env' ),
 			{
-				corejs: 2,
+				corejs: 3.6,
```

Comment:
are there docs that show this is a supported value? The babel docs seem to want to make this `2` or `3`. Having it be a float seems odd, I'd expect a semver string. https://babeljs.io/docs/en/babel-preset-env#corejs

Reply:
I was following https://github.com/zloirock/core-js/blob/master/docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md#babelpreset-env , but there it appears to need to be a string.

Looking at the code, this *should* work, but given the lack of mention in the official docs, it's probably best to switch to just `3` here.

Done, thanks for the suggestion!

Reply:
I had this same question, and also checked this in the Babel code. The logic is to use CoreJS 2 if the value is `2`, and to use CoreJS 3 if the value is anything else, including `3.6` 🙂